features/if.yml

```
@@ -0,0 +1,5 @@
+name: if()
+description: The if() function is an inline conditional value function that selects a value based on a set of conditions.
```

```suggestion
description: The `if()` CSS function is an inline conditional value that returns a value based on a set of conditions.
```

I'm on the fence about `returns` vs `selects`. In the context of CSS, we've used `selects` as the verb describing the element that a selector is selecting. We use `return` on `min()` and `max()` so I think it makes sense here.

I was also going back and forth on whether or not `is an inline conditional value that` is important here, but I think it's helpful to differentiate from flow control.

---

Great suggestions. I believe I went with selects because of "conditional value selection" from the spec. Having used it and considering how I would talk about it in conversation, I would also be more likely to use `returns` over `selects`.